src/i18n/ja-JP/common.yml

```diff
@@ -0,0 +1,986 @@
+
+          - Avoid duplicate responses
+
+          - Give you access to your data
+
+          - Save your session as you go
+
+          - Notify you when results are live
+      t: >
+          以下の理由からアカウントの作成をお願いしています。
+
+          - 回答の重複を避けたい
+
+          - あなたが自身のデータへアクセスできるようにしたい
+
+          - ページ離脱時にセッションを保存したい
+
+          - 結果が公開されたときにあなたに伝えたい
+    - key: general.survey_closed
+      t: >
+          Sorry, the survey is now closed! You can still review your data but you won't be able to modify it.
+      t: >
+          ごめんなさい！アンケートは終了しました。あなたが送信したデータの確認はできますが、変更はできません。
+    - key: general.survey_read_only
+      t: >
+          You are currently viewing this survey in read-only mode.
+      t: >
+          アンケートを読み込み専用モードで見ています。
+    - key: general.survey_read_only_back
+      t: >
+          <a href="${link}">Go back</a> to the main survey page to start or resume a session.
+      t: >
+          <a href="${link}">メインページへ戻る</a>。アンケートへの回答や、回答中のアンケートを再開できます。
+
+    # surveys
+    - key: general.open_surveys
+      t: Open Surveys
+      t: 実施中のアンケート
+    - key: general.no_open_surveys
+      t: No surveys are currently open.
+      t: 実施中のアンケートはありません。
+    - key: general.closed_surveys
+      t: Closed Surveys
+      t: 終了したアンケート
+    - key: general.no_open_surveys
+      t: No surveys are currently closed.
+      t: 終了したアンケートはありません。
+    - key: general.preview_surveys
+      t: Preview Surveys
+      t: アンケートのプレビュー
+    - key: general.no_preview_surveys
+      t: No surveys available to preview.
+      t: プレビューできるアンケートはありません。
+    - key: general.start_survey
+      t: Start Survey »
+      t: アンケートに回答する »
+    - key: general.continue_survey
+      t: Continue Survey »
+      t: アンケートの回答を続ける »
+    - key: general.review_survey
+      t: Review Survey »
+      t: アンケートの回答を確認する »
+    - key: general.preview_survey
+      t: Preview Survey »
+      t: アンケートの回答をプレビュー »
+    - key: general.finish_survey
+      t: Finish Survey »
+      t: アンケートを終了する »
+    - key: general.survey_closed
+      t: This survey is now closed.
+      t: このアンケートは終了しました。
+    - key: general.survey_results
+      t: View Survey Results »
+      t: アンケートの結果を見る »
+    - key: general.surveys_available_languages
+      t: 'Surveys available in:'
+      t: アンケートは次の言語でも見られます：
+    - key: general.help_us_translate
+      t: Help us translate the surveys »
+      t: アンケートの翻訳に協力してください »
+    - key: general.translators
+      t: Translators
+      t: 翻訳者
+    - key: general.translation_help
+      t: Translation Help
+      t: 翻訳のヘルプ
+
+    # footer
+    - key: general.privacy_policy
+      t: Privacy Policy
+      t: プライバシーポリシー
+    - key: general.leave_issue
+      t: Questions? Found a bug? <a href="${link}" target="_blank">Leave an issue</a>.
+      t: 質問があったり、バグを見つけたときは<a href="${link}" target="_blank">issueを立ててください</a>。
+    - key: general.emoji_icons
+      t: Emoji icons by <a target="_blank" rel="noreferrer" href="https://icons8.com">Icons8</a>
```

Comment:
I intentionally left this as it is. We don't translate small texts or credits very much. Doing so would make the site look machine-translated, and in case of Japanese, the quality is kinda meh :(